src/openrct2/ride/Vehicle.h

```diff
@@ -221,7 +221,7 @@ struct Vehicle : SpriteBase
     union
     {
         int16_t track_direction; // (0000 0000 0000 0011)
-        int16_t track_type;      // (0000 0011 1111 1100)
+        int16_t track_type;      // (1111 1111 1111 1100)
```

Comment:
for a follow on PR we should remove this union make it so everything access's via the getters.